padevents/padevents.py

```diff
@@ -460,20 +460,56 @@ async def aep_s_channel(self, ctx, key, channel: discord.TextChannel):
         await ctx.tick()
 
     @aep_set.command(name="redchannel")
-    async def aep_s_redchannel(self, ctx, key, channel: discord.TextChannel):
+    async def aep_s_redchannel(self, ctx, key, channel: discord.TextChannel=None):
         """Sets channel to ping when event is red"""
+        if channel is None:
+            if key not in await self.config.guild(ctx.guild).pingroles():
+                await ctx.send("That key does not exist.")
+                return
+            if not await get_user_confirmation(ctx, "Are you sure you want to remove the red channel for AEP `{}`?".format(key)):
+                await send_cancellation_message(ctx, "No action was taken. You can set a channel with `[p]aep set redchannel {} #channel_name`"
+                                                .format(key))
+                return
+            else:
+                await self.aepchange(ctx, key, 'channels', lambda x: [None, x[1], x[2]])
+                await send_confirmation_message(ctx, "Okay, I removed the red channel for the AEP `{}`".format(key))
+                return
         await self.aepchange(ctx, key, 'channels', lambda x: [channel.id, x[1], x[2]])
         await ctx.tick()
 
     @aep_set.command(name="bluechannel")
-    async def aep_s_bluechannel(self, ctx, key, channel: discord.TextChannel):
+    async def aep_s_bluechannel(self, ctx, key, channel: discord.TextChannel=None):
         """Sets channel to ping when event is blue"""
+        if channel is None:
```

Comment:
Make a helper function to do this cos it's the same code copied 3x, you can pass the color as a param

Reply:
If we make the color a StarterGroup enum (see: https://github.com/TsubakiBotPad/tsutils/blob/master/tsutils/enums.py#L24) then it should be easier to figure out how to make the lambdas work